README.md

```diff
@@ -747,6 +748,39 @@ pipeline.notifyOnFailure(PipelineNotification.chime({
 }));
 ```
 
+## ECR Mirror
+
+Builds commonly use Docker images, and these typically come from DockerHub. In fact, delivlib defaults its build
+image to `jsii/superchain`. However, DockerHub has throttles in place for the volume of unauthenticated and
+authenticated pulls. This can cause CodeBuild jobs that run frequently to fail from DockerHub's throttling.
+
+The `EcrMirror` construct can be used to synchronize, on a specific schedule, Docker images between DockerHub and
+a local ECR registry in the AWS account.
+
+```ts
+new EcrMirror(this, 'RegistrySync', {
+  images: [
+    MirrorSource.fromDockerHub('jsii/superchain'),
+    MirrorSource.fromDockerHub('python:3.6'),
+  ],
```

Comment:
Optional: since the type of content is `MirrorSource`, then should this key be called `sources:`?

Reply:
Good call.